include/Analyzer.h

```diff
@@ -155,7 +166,7 @@ namespace insur {
                                const std::vector<double>& momenta,
                                const std::vector<double>& triggerMomenta,
                                const std::vector<double>& thresholdProbabilities,
-			       bool isPixel,
+			       bool isOuterPixel,
```

Comment:
Hmm you are right, this was probably accidentally written when looking for word 'Outer'.
This has absolutely no effect though, since in this context only the type matters.
I just fixed this typo though ;)